image/vision_transformer/data_parallel/train.py

```diff
@@ -19,17 +19,15 @@ def build_dali_train():
     root = os.environ['DATA']
     train_pat = os.path.join(root, 'train/*')
     train_idx_pat = os.path.join(root, 'idx_files/train/*')
-    return DaliDataloader(
-        sorted(glob.glob(train_pat)),
-        sorted(glob.glob(train_idx_pat)),
-        batch_size=gpc.config.BATCH_SIZE,
-        shard_id=gpc.get_local_rank(ParallelMode.DATA),
-        num_shards=gpc.get_world_size(ParallelMode.DATA),
-        gpu_aug=gpc.config.dali.gpu_aug,
-        cuda=True,
-        mixup_alpha=gpc.config.dali.mixup_alpha,
-        randaug_num_layers=2
-    )
+    return DaliDataloader(sorted(glob.glob(train_pat)),
+                          sorted(glob.glob(train_idx_pat)),
+                          batch_size=gpc.config.BATCH_SIZE,
+                          shard_id=gpc.get_local_rank(ParallelMode.DATA),
+                          num_shards=gpc.get_world_size(ParallelMode.DATA),
+                          gpu_aug=gpc.config.dali.gpu_aug,
+                          cuda=True,
+                          mixup_alpha=gpc.config.dali.mixup_alpha,
+                          randaug_num_layers=2)
```

Comment:
Did you apply pre-commit formatting?

Reply:
yes，I think the above changes is done by pre-commit.